packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx

```diff
@@ -131,71 +254,59 @@ const TableCalculationModal: FC<Props> = ({
                 }}
             >
                 <DialogBody className={Classes.DIALOG_BODY}>
-                    <Input
+                    <TextInput
+                        mb="sm"
                         label="Name"
                         name="name"
                         disabled={isDisabled}
-                        rules={{
```

Comment:
we used to block table calculations with the same name, but now we have a function that allows having the same name, but assigns a different autoincremental id to each TC, so luckily we don't need this validation rule anymore